src/python/chrome/build_info.py

```diff
@@ -95,25 +119,12 @@ def get_production_builds_info_from_cd(platform):
   Platform is e.g. ANDROID, LINUX, MAC, WINDOWS.
   """
   builds_metadata = []
-  chromiumdash_platform = _convert_platform_to_chromiumdash_platform(platform)
-  query_url = BUILD_INFO_URL_CD.format(platform=chromiumdash_platform)
-
-  build_info = utils.fetch_url(query_url)
-  if not build_info:
-    logs.log_error('Failed to fetch build info from %s' % query_url)
-    return []
-
-  try:
-    build_info_json = json.loads(build_info)
-    if not build_info_json:
-      logs.log_error('Empty response from %s' % query_url)
-      return []
-  except Exception:
-    logs.log_error('Malformed response from %s' % query_url)
-    return []
-
+  build_info_json = _fetch_releases_from_chromiumdash(platform)
   for info in build_info_json:
     build_type = info['channel'].lower()
+    if build_type == 'extended':
+      build_type = 'extended_stable'
```

Comment:
Converting `extended` to `extended_stable` to make sure its representation is universally the same in this repro, instead of mixing two format for ES.